### PR TITLE
Creating place page methods

### PR DESCRIPTION
**Pull request by RoxVer**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/result of registration.jsp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
```
Comment (Sabfir) [inline src/main/webapp/WEB-INF/views/result of registration.jsp:1]: Plz, change the name of the file to the registrationResult.jsp
Refer to the name convention
http://stackoverflow.com/questions/14245535/file-naming-convention-for-jsp

File: src/main/webapp/resources/app/register/register.template.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      
+      <p class="message">Already registered? <a href="#">Sign In</a></p>
+    </form>
+  </div>
+</div>
+ 
+ 
+ 
+ 
+
```
Comment (Sabfir) [inline src/main/webapp/resources/app/register/register.template.html:21]: Plz, delete redundant empty lines.
It's good practice to leave one empty line at the end of the file

File: src/main/webapp/WEB-INF/views/register.jsp [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<%@ taglib uri="http://www.springframework.org/tags/form" prefix="form" %>
```
Comment (Sabfir) [inline src/main/webapp/WEB-INF/views/register.jsp:1]: Plz, delete this file, we have no need in new jsp pages.
Create angular components instead

File: src/main/java/ua/softserve/rv_018/greentourism/controller/MainController.java [hunk 4]
```diff
@@ -1,89 +1,101 @@
...
+
+		model.setViewName("login");
+
+		return model;
+	}
+	
+	
+	@RequestMapping(value = "/register")
+	public String registerPage(Model model) {
+
```
Comment (Sabfir) [inline src/main/java/ua/softserve/rv_018/greentourism/controller/MainController.java:64]: Do we really need this controller method?
Where do you use it?
If not, plz, delete it.

**Comment by Sabfir** [comment]
The branch name is creating-place-page.
Where is implementation?
I can't see any related with this task code.

File: src/main/webapp/WEB-INF/views/result of registration.jsp (hunk 1 above)
Comment (RoxVer) [inline src/main/webapp/WEB-INF/views/result of registration.jsp:1]: oh, I didn't plan to commit this file yet.

Comment (RoxVer) [inline src/main/webapp/WEB-INF/views/result of registration.jsp:1]: I used one plugin to commit, apparently I haven't figured out how to use it yet.

**Pull request closed without merging**